lib/tasks/import_events.rake

```diff
@@ -0,0 +1,62 @@
+require 'csv'
+
+namespace :import do
+  desc "import CSV data into database"
+  task :events, [:csv_file] => [:environment] do |t, args|
+
+    import_csv_file(args.csv_file)
+  end
+
+  def import_csv_file file
+    csv = File.open(file, encoding: "utf-8")
+    csv_headers = File.readlines(csv).first.split(",")
+
+    event_hash = {
+      title: csv_headers[0],
+      start_date: csv_headers[1],
+      end_date: csv_headers[2],
+      location: csv_headers[3],
+      organisers: csv_headers[4],
+      summary: csv_headers[5],
+      relevance: csv_headers[6],
+      outputs: csv_headers[7],
+      category_first: csv_headers[8],
+      category_second: csv_headers[9].chomp
+    }
+
+    CSV.parse(csv, headers: true, encoding: "utf-8") do |row|
+      csv_event_row = row.to_hash
+      event_row_hash = {}
+
+      event_hash.keys.each do |key|
+        if [:outputs, :summary, :category_first].include? key
+          event_row_hash[key] = csv_event_row[event_hash[key]]&.strip || "Empty"
+        elsif [:start_date, :end_date].include? key
+          event_row_hash[key] = csv_event_row[event_hash[key]]&.strip || nil
```

Comment:
I would like to have this "Empty" or "nil" standardised for all the values, but it might affect the filters etc. and it also depends on the file we are given. I'll merge it for now but we should keep an eye on this as I believe there is room for improvement :)

Reply:
Ok cool thanks @Levia !